lib/components/molecules/Skeleton/Skeleton.story.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+import { defaultConfig, circleConfig } from './Skeleton.mock';
+
+// Import Styled Component to showcase variations
+import Skeleton from '.';
+
+storiesOf('Molecules/Skeleton', module)
+  .addParameters({ jest: ['Skeleton', 'SkeletonVanilla'] })
+  .add('Skeleton', () => (
```

Comment:
We need to create few better examples. May be take few components such as accordion, Tabs, list and apply this Skelton there.